mix.exs

```diff
@@ -23,13 +24,60 @@ defmodule Floki.Mixfile do
   end
 
   defp deps do
+    # Needed to avoid installing unnecessary deps on travis
```

Comment:
If we don't need Travis, I think we can remove this too, right?

Reply:
Applies to Gitlab Actions as well, I just wrote this comment when I didn't know you had it :)